src/Nethermind/Nethermind.JsonRpc/JsonRpcProcessor.cs

```diff
@@ -33,11 +33,11 @@ namespace Nethermind.JsonRpc
 {
     public class JsonRpcProcessor : IJsonRpcProcessor
     {
-        private IJsonRpcService _jsonRpcService;
-        private IJsonSerializer _jsonSerializer;
+        public IJsonRpcService JsonRpcService { get; }
```

Comment:
Is it public for testing? Then InternalsVisibleTo would suffice.

Reply:
Removed